agent/ansible/Makefile

```diff
@@ -0,0 +1,27 @@
+# How to talk to Galaxy
+apiserver = https://galaxy.ansible.com/api/
+apikey = ${APIKEY}
+
+# Collection info
+namespace = pbench
+collection = agent
+version = 1.0.2
```

Comment:
Should this track the pbench version?

Reply:
So if that is the case, any way this `Makefile` can derive the version from the `.yml` file to avoid having to keep the version number in two places?

Reply:
Added a yamlconf script to get things out of a yaml file and invoked that from the Makefile.